quipucords/api/source/serializer.py

```diff
@@ -74,7 +63,7 @@ class SourceSerializer(NotEmptySerializer):
     port = IntegerField(required=False, min_value=0, allow_null=True)
     hosts = JSONField(required=True)
     exclude_hosts = JSONField(required=False)
-    options = SourceOptionsSerializer(required=False, many=False)
+
     credentials = CredentialsField(many=True, queryset=Credential.objects.all())
```

Comment:
I have a PTSD everytime I see something that COULD be paginated in a single response.

Maybe we could (in subsequent PRs) force a limit on the backend to not allow more than an arbitrary number of credentials to be attached to a source. If not possible, then remove this field (sorry UI/CLI who will be making some extra requests).

Reply:
I'm being admittedly paranoid with this one. we can deal with it later.